database/src/main/resources/db/migration/V1.89.0__RHCLOUD-29487_add_gateway_certificate_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE gateway_certificate (
+    id UUID NOT NULL,
+    application_id UUID NOT NULL,
+    certificate_data text NOT NULL,
+    environment text NOT NULL,
+    CONSTRAINT pk_gateway_certificate_notification PRIMARY KEY (id)
+);
+
```

Comment:
Could you please add a `unique` constraint for the `bundle/app/subject_dn` triplet?